.github/workflows/integration_test.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           docker tag $imageId localhost:32000/gunicorn:latest
           docker push localhost:32000/gunicorn:latest
       - name: Run integration tests
-        run: tox -e integration -- --model testing --gunicorn-image localhost:32000/gunicorn:latest --statsd-prometheus-exporter-image prom/statsd-exporter
+        run: tox -e integration -- --model testing --gunicorn-image localhost:32000/gunicorn:latest --statsd-prometheus-exporter-image prom/statsd-exporter --machine-controller-name localhost --influx-model-name influxmodel
```

Comment:
The `statsd-prometheus-exporter-image` should be already picked up by the tests from the metadata definition, AFAIK